src/interfaces/IHooks.sol

```diff
@@ -103,7 +104,7 @@ interface IHooks {
         PoolKey calldata key,
         IPoolManager.SwapParams calldata params,
         bytes calldata hookData
-    ) external returns (bytes4, int128);
+    ) external returns (bytes4, BeforeSwapDelta);
```

Comment:
nit: should probably update the @return comment for this

Reply:
think this still needs updating